src/main/resources/templates/in-app/deferral/v1.0.0.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>Deferral</title>
+</head>
+<body>
+<h1>In-App Notification</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">Deferral</th:block>
+<h2>Content</h2>
+<th:block th:fragment="content">
+  <p>
+    This notification pertains to the following programme
+    <th:block th:text="${not #strings.isEmpty(programmeName)} ? ${programmeName} : _"
+    >(programme name missing)</th:block
+    >
+    starting on
+    <th:block th:text="${startDate} ? |${#temporals.format(startDate, 'dd MMMM yyyy')}| : _"
+    >(start date missing)</th:block
+    >.
+  </p>
+  <p>
+    As per the <a href="https://www.copmed.org.uk/publications/gold-guide">Gold Guide</a>, doctors
+    in training that wish to defer the start date of their specialty training programme can normally
+    only do so on statutory grounds i.e., personal ill health or maternity/paternity/adoption leave.
+  </p>
+  <p>
+    Doctors in training wishing to defer must apply. Please follow your local office process.
+    <th:block: th:switch="${localOfficeContactType}">
+      <a th:case="url" th:href="${localOfficeContact}" th:text="${localOfficeContact}"></a>
+      <a
+        th:case="email"
+        th:href="|mailto:${localOfficeContact}|"
+        th:text="${localOfficeContact}"
+      ></a>
+      <th:block th:case="*"
+      >Contact your local deanery office for information on the deferral process.</th:block
+      >
+    </th:block:>
+  </p>
```

Comment:
I've refactored the equivalent paragraph on the LTFT notification so that it reads better when there is no contact available, check with Naz whether an equivalent change is wanted here.